drinfomon.lic

```diff
@@ -544,7 +544,7 @@ end
 def find_pcs(room_players)
   room_players.sub(/ and (.*)$/) { ", #{Regexp.last_match(1)}" }
               .split(', ')
-              .map { |obj| obj.sub(/ (who|whose body) (has|is|appears|glows) .+/, '').sub(/ \(.+\)/, '') }
+              .map { |obj| obj.sub(/ (who|whose body)? ?(has|is|appears|glows) .+/, '').sub(/ \(.+\)/, '') }
```

Comment:
🧹  Stylistic suggestions
```suggestion
              .map { |obj| obj.sub(/ (?:who|whose body)?\s?(?:has|is|appears|glows) .+/, '').sub(/ \(.+\)/, '') }
```

Reply:
Mahtra just mentioned this in Discord. Good stylistic point. Should also tag that leading space with a `\s` I assume.